src/constants/translations/en/tutor-home-page.json

```diff
@@ -0,0 +1,46 @@
+{
+    "findStudentBlock": {
```

Comment:
i don`t see you use all those translations in your component

Reply:
> i don`t see you use all those translations in your component

yeah, removed it